quarkus-opentelemetry-exporter-sentry/integration-tests/src/test/java/io/quarkiverse/opentelemetry/exporter/it/SentryTest.java

```diff
@@ -0,0 +1,73 @@
+package io.quarkiverse.opentelemetry.exporter.it;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.testcontainers.shaded.org.awaitility.Awaitility.await;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
+import com.github.tomakehurst.wiremock.verification.LoggedRequest;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class SentryTest {
+
+    public static final int HTTP_PORT_NUMBER = 53602; // See application.properties file
+    private WireMockServer wireMockServer;
+
+    @BeforeEach
+    public void startWireMock() {
+        WireMockConfiguration wireMockConfiguration = new WireMockConfiguration().port(HTTP_PORT_NUMBER);
+        wireMockServer = new WireMockServer(wireMockConfiguration);
+        wireMockServer.start();
+    }
+
+    @AfterEach
+    public void stopWireMock() {
+        wireMockServer.stop();
+    }
+
+    @Test
+    void connectionTest() throws InterruptedException {
+
+        wireMockServer.stubFor(
+                any(urlMatching(".*"))
+                        .withPort(HTTP_PORT_NUMBER)
+                        .willReturn(aResponse().withStatus(200)));
+
+        given()
+                .contentType("application/json")
+                .when().get("/direct")
+                .then()
+                .statusCode(200)
+                .body("message", equalTo("Direct trace"));
+
+        await()
+                .atMost(Duration.ofSeconds(10))
+                .until(telemetryDataContainTheHttpCall(wireMockServer));
+
+        // Non regression test for https://github.com/Azure/azure-sdk-for-java/issues/41040
+        Thread.sleep(10_000);
+        List<LoggedRequest> telemetryExport = wireMockServer.findAll(postRequestedFor(urlEqualTo("/")));
+        List<String> requestBodies = telemetryExport
+                .stream()
+                .map(request -> new String(request.getBody())).toList();
+        assertThat(requestBodies).hasSize(1);
+        assertThat(requestBodies.get(0)).contains("sentry.java");
```

Comment:
Is it possible to create tests asserting actual tracings, metrics and logs data?

Reply:
@brunobat I worked a little bit on integration tests, WDYT ?

I also added some doc